### PR TITLE
add partials (don't merge)

### PR DESCRIPTION
**Pull request by roberthopman**
- [ ]  add partials
- [ ]  add sasspower to files for mixin
- [ ]  add boostrap to files (work in progress)

### CODE REVIEW

File: config/secrets.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#  Make sure the secret is at least 30 characters and all random,
+#  no regular words or you'll be exposed to dictionary attacks.
+#  You can use `rake secret` to generate a secure secret key.
+#
+#  Make sure the secrets in this file are kept private
+#  if you're sharing your code publicly.
+#
+
+development:
+  secret_key_base: 1234567890123456789012345678901a
```
Comment (salielim) [inline config/secrets.yml:15]: is this sensitive info that shouldn't be pushed to a public repo?

File: app/assets/stylesheets/custom.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+@import "microposts";
+@import "users";
+@import "forms";
+@import "sidebar";
+@import "footer";
+@import "header";
+@import "sidebar";
+@import "typography";
+@import "universal";
+@import "sasspower";
```
Comment (salielim) [inline app/assets/stylesheets/custom.scss]: curious why it is named `sasspower` where did you get the name from

File: app/assets/stylesheets/_forms.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
+@import "bootstrap";
```
Comment (salielim) [inline app/assets/stylesheets/_forms.scss]: have imported bootstrap in `app/assets/stylesheets/custom.scss`, have to do this again for every file?

File: config/secrets.yml (hunk 1 above)
Comment (roberthopman) [reply config/secrets.yml:15]: It's just a random string that is longer than 30 characters

File: app/assets/stylesheets/custom.scss (hunk 2 above)
Comment (roberthopman) [reply app/assets/stylesheets/custom.scss]: I'll rename it to the correct thing (mixin)

File: app/assets/stylesheets/_forms.scss (hunk 3 above)
Comment (roberthopman) [reply app/assets/stylesheets/_forms.scss]: Right now yes. But I should put it in a directory just like you did. To prevent this.

**Pull request closed without merging**